app/src/main/java/com/eventyay/organizer/core/auth/start/StartFragment.java

```diff
@@ -46,24 +50,39 @@
     private StartFragmentBinding binding;
     private Validator validator;
     private SharedViewModel sharedViewModel;
+    private SharedPreferences sharedPreferences;
+    private SharedPreferences.Editor editor;
+    AlertDialog warningAlertDialog;
 
     public static StartFragment newInstance() {
         return new StartFragment();
     }
 
+    @SuppressLint("WrongConstant")
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
         binding = DataBindingUtil.inflate(inflater, R.layout.start_fragment, container, false);
         startFragmentViewModel = ViewModelProviders.of(this, viewModelFactory).get(StartViewModel.class);
         sharedViewModel = ViewModelProviders.of(getActivity()).get(SharedViewModel.class);
         validator = new Validator(binding);
+        sharedPreferences = requireActivity().getSharedPreferences(getResources().getString(R.string.warning_preference_key), Context.MODE_APPEND);
+        editor = sharedPreferences.edit();
         return binding.getRoot();
     }
 
     @Override
     public void onStart() {
         super.onStart();
-
+        if (!sharedPreferences.getBoolean(getResources().getString(R.string.warning_preference_key), false)){
```

Comment:
Codacy found an issue: ['if' is not followed by whitespace.](https://app.codacy.com/manual/fossasia/open-event-orga-app/pullRequest?prid=6423984)